views/random-beer.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="random-beers-wrap">
+    <div class="random-beer-container">
+        {{> beerPartial randomBeer}}
+        {{#with randomBeer}}
```

Comment:
You could include the beerPartial to be included within the "with", in such case you could have to change `{{> beerPartial randomBeer}}` to `{{> beerPartial this}}`